tests/unit/amo/components/TestSearchPage.js

```diff
@@ -1,9 +1,18 @@
-import { mapStateToProps } from 'amo/components/SearchPage';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { SearchPageBase, mapStateToProps } from 'amo/components/SearchPage';
+import Search from 'amo/components/Search';
 import { CLIENT_APP_ANDROID } from 'core/constants';
 import { dispatchClientMetadata } from 'tests/unit/amo/helpers';
+import { getFakeI18nInst } from 'tests/unit/helpers';
 
 
 describe(__filename, () => {
+  function render(props = {}) {
+    return shallow(<SearchPageBase {...{ ...props, i18n: getFakeI18nInst() }} />);
```

Comment:
This should use `shallowUntilTarget(<SearchPage />, SearchPageBase)` instead so we can test the composed component. `shallowUntilTarget` is in `tests/unit/helpers`.